modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/styles/components/_activation-keys.scss

```diff
@@ -34,18 +34,22 @@
 	border-right-color: transparent;
 }
 
-.dropdown button:focus {
-	box-shadow: none;
-}
-
 .cp-activation-key-icon {
 	border: 1px solid var(--color-neutral-2);
 	box-shadow: none;
-	left: 641px;
+	left: 648px !important;
 	min-width: fit-content;
-	top: 276px;
-	ul li span {
+	top: 276px !important;
+	ul li button {
 		padding: 0.75rem;
 		padding-right: 68px !important;
 	}
 }
+
+.dropdown button:focus {
+	box-shadow: none;
+}
+
+.cp-activation-status-container .card-body {
+	padding-right: 13px !important;
```

Comment:
Same as discuss about `!important` usage.
Maybe we can change to use the `pr-3` class, if possible.

There is no problem with `1px` difference.

https://github.com/liferay/liferay-frontend-projects/blob/master/guidelines/css/when_to_use_css_utility_classes.md#2-the-important-attribute